### PR TITLE
refactor(chat): change messages markdown

### PR DESCRIPTION
**Pull request by AlexZakablukov**
**What this PR does** 📖
change messages markdown

**Which issue(s) this PR fixes** 🔨
[AP-1454](https://satellite-im.atlassian.net/browse/AP-1454)

**Special notes for reviewers** 🗒️

**Additional comments** 🎤

### CODE REVIEW

**Review by phillsatellite — APPROVED** [review]

File: libraries/Enums/types/messaging-types.ts [hunk 1]
```diff
@@ -8,6 +8,7 @@ export enum MessagingTypesEnum {
   GLYPH = 'glyph',
   DIVIDER = 'divider',
   STRING = 'string',
+  MESSAGEELEMENT = 'messageElement',
```
Comment (molimauro) [inline libraries/Enums/types/messaging-types.ts:11]: Can be removed

File: types/messaging.d.ts [hunk 2]
```diff
@@ -42,4 +45,16 @@ export type Group = {
   messages: Array<UIMessage> | null
 }
 
+// TO DO: MAURO CHECK
```
Comment (molimauro) [inline types/messaging.d.ts:48]: Can be removed

File: types/messaging.d.ts [hunk 3]
```diff
@@ -42,4 +45,16 @@ export type Group = {
...
+export type MessageElement = {
+  id: string
+  at: number
+  type: MessagingTypesEnum.MESSAGEELEMENT
+  from: string
+  to: string
+  message: UIMessage | null
+}
+
+export type ChatList = Array<MessageElement | Divider>
```
Comment (molimauro) [inline types/messaging.d.ts:60]: Can be removed

**Comment by AlexZakablukov** [comment]
fixed

**Review by phillsatellite — APPROVED** [review]

**Review by molimauro — APPROVED** [review]

**Review by stavares843 — APPROVED** [review]

**Pull request merged**